packages/react-kit/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # @pantheon-systems/react-kit
 
-## 0.2.0-canary.0
+## 0.2.0
```

Comment:
This seems weird to me but not blocking. Other CHANGELOGS have both canary and main releases listed, I wouldn't exepect it to overwrite the canary.